packages/ckeditor5-table/tests/tablelayout/tablelayoutediting.js

```diff
@@ -0,0 +1,975 @@
+							'<tableCell><paragraph>2</paragraph></tableCell>' +
+							'<tableCell><paragraph>3</paragraph></tableCell>' +
+						'</tableRow>' +
+						'<tableColumnGroup>' +
+							'<tableColumn columnWidth="28.59%"></tableColumn>' +
+							'<tableColumn columnWidth="19.93%"></tableColumn>' +
+							'<tableColumn columnWidth="51.48%"></tableColumn>' +
+						'</tableColumnGroup>' +
+					'</table>'
+				);
+			} );
+		} );
+	} );
+
+	describe( 'clipboard pipeline', () => {
+		it( 'should not crash the editor if there is no clipboard plugin', async () => {
+			await editor.destroy();
+
+			editor = await ClassicTestEditor.create( editorElement, {
+				plugins: [ TableEditing, TableLayoutEditing ]
+			} );
+
+			expect( editor.plugins.get( 'TableLayoutEditing' ) ).to.be.instanceOf( TableLayoutEditing );
+		} );
+
+		describe( 'pasting content', () => {
+			it( 'should preserve table type if paste within the same editor', () => {
+				const dataTransferMock = createDataTransfer( {
+					'application/ckeditor5-editor-id': editor.id,
+					'text/html': '<table class="table layout-table"><tbody><tr><td>Foo</td></tr></tbody></table>'
+				} );
+
+				view.document.fire( 'paste', {
+					dataTransfer: dataTransferMock,
+					preventDefault: () => {},
+					stopPropagation: () => {},
+					method: 'paste'
+				} );
+
+				expect( getModelData( model ) ).to.equal(
+					'[<table tableType="layout">' +
+						'<tableRow>' +
+							'<tableCell>' +
+								'<paragraph>Foo</paragraph>' +
+							'</tableCell>' +
+						'</tableRow>' +
+					'</table>]'
+				);
+
+				expect( editor.getData() ).to.equal(
+					'<table class="table layout-table" role="presentation">' +
+						'<tbody>' +
+							'<tr><td>Foo</td></tr>' +
+						'</tbody>' +
+					'</table>'
+				);
+			} );
+
+			it( 'should preserve table type if paste from the another editor', () => {
+				const dataTransferMock = createDataTransfer( {
+					'application/ckeditor5-editor-id': 'other-editor',
+					'text/html': '<table class="table layout-table"><tbody><tr><td>Foo</td></tr></tbody></table>'
+				} );
+
+				view.document.fire( 'paste', {
+					dataTransfer: dataTransferMock,
+					preventDefault: () => {},
+					stopPropagation: () => {},
+					method: 'paste'
+				} );
+
+				expect( getModelData( model ) ).to.equal(
+					'[<table tableType="layout">' +
+						'<tableRow>' +
+							'<tableCell>' +
+								'<paragraph>Foo</paragraph>' +
+							'</tableCell>' +
+						'</tableRow>' +
+					'</table>]'
+				);
+
+				expect( editor.getData() ).to.equal(
+					'<table class="table layout-table" role="presentation">' +
+						'<tbody>' +
+							'<tr><td>Foo</td></tr>' +
+						'</tbody>' +
+					'</table>'
+				);
+			} );
+
+			it( 'should convert to content table if paste from external', () => {
+				const dataTransferMock = createDataTransfer( {
+					'text/html': '<figure><table><tbody><tr><td>Foo</td></tr></tbody></table></figure>'
```

Comment:
```suggestion
					'text/html': '<table><tbody><tr><td>Foo</td></tr></tbody></table>'
```

Reply:
@niegowski I restored clipboard handler in this https://github.com/ckeditor/ckeditor5/pull/17931/commits/f383923035c5b64c551e6314c24d29bb06b4a2ed. Let me know if it fixes the issue.